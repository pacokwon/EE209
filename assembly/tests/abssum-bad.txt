|
p
3
|
p
q
