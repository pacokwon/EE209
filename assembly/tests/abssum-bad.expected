dc: stack empty
dc: stack empty
dc: stack empty
3
3
